specification/azsadmin/resource-manager/keyvault/Microsoft.KeyVault.Admin/2017-02-01-preview/Quotas.json

```diff
@@ -0,0 +1,116 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2017-02-01-preview",
+        "title": "KeyVaultManagementClient",
+        "description": "The Admin KeyVault Management Client."
+    },
+    "host": "management.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.KeyVault.Admin/locations/{location}/quotas": {
+            "get": {
+                "x-ms-examples": {
+                    "Get a list of all quota objects for KeyVault at a location.": {
+                        "$ref": "./examples/Quota/List.json"
+                    }
+                },
+                "description": "Get a list of all quota objects for KeyVault at a location.",
+                "tags": [
+                    "KeyVault"
+                ],
+                "operationId": "Quotas_List",
+                "parameters": [
+                    {
+                        "$ref": "KeyVault.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "KeyVault.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "KeyVault.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/QuotaList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Quota": {
+            "description": "KeyVault Quota.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Properties for keyvault quotas.",
+                    "value": null,
```

Comment:
Syntax error: please remove "value".

Reply:
Done.  I don't get a syntax error locally, oav and autorest seem to work fine with it.